db/languages.yml

```diff
@@ -138,3 +138,17 @@ j:
     j8:
       :name: "J 8.0.3"
       :interpreter: /usr/bin/ijconsole-8.0.3
+javascript:
+  :name: JavaScript
+  :lexer: javascript
+  :compiled: no
+  :interpreted: yes
+  :current: v8_8
+  :extension: .js
+  :source_filename: program
+  :interpreter_command: '%{interpreter} %{source}'
+  :variants:
+    v8_8:
+      :name: "JavaScript V8"
```

Comment:
Could be good to be more explicit with `V8 Engine 8.1`. Otherwise it looks like JavaScript version 8.

Reply:
Will change this to `JavaScript (V8 8.1)` for now.